src/main/java/Duke.java

```diff
@@ -1,10 +1,179 @@
+                            + " Now you have " + listOfTasks.size() + (listOfTasks.size() <= 1 ? " task in the list." : " tasks in the list.")
+                            + "\n____________________________________________________________\n");
+                }
+
+                whatToDoCall = whatToDo.nextLine().trim().toLowerCase();
+                command = whatToDoCall.split(" ", 2);
+            }
+
+            try {
+                File directory = new File("./data/");
+
+                if (!directory.exists()) {
+                    directory.mkdirs();
+                }
+
+                saveToFile("./data/duke.txt", stringToSave(listOfTasks));
+            } catch (IOException e) {
+                System.out.println("Error while saving, please try again!");
+            }
+        } catch (DukeException exception) {
+            System.out.println(exception.getMessage());
+        }
+
+        System.out.println("____________________________________________________________\n"
+                + " Bye. Hope to see you again soon!\n - Your Only Friend\n"
+                + "____________________________________________________________\n");
+
+
+    }
+
+    public static void load(ArrayList<Task> listOfTasks) {
+
+        try {
+            File file = new File("./data/duke.txt");
+            Scanner whatToDo = new Scanner(file);
+
+            while (whatToDo.hasNext()) {
+                String readText = whatToDo.nextLine();
+                char taskType = readText.charAt(1);
+                char doneOrNot = readText.charAt(5);
+                String description = readText.substring(9);
+
+                Task task = null;
+
+                if (taskType == 'T') {
+                    task = new ToDo(description);
+                    listOfTasks.add(task);
+                } else if (taskType == 'D') {
+                    try {
+                        task = new Deadline(description);
+                        listOfTasks.add(task);
+                    } catch (DukeException e) {
+                        System.out.println(e.getMessage());
+                    }
+                } else {
+                    try {
+                        task = new Event(description);
+                        listOfTasks.add(task);
+                    } catch (DukeException e) {
+                        System.out.println(e.getMessage());
+                    }
+                }
+
+                if (doneOrNot == '1') {
+                    task.markComplete();
+                }
+
+            }
+
+        } catch (FileNotFoundException e) {
+                System.out.println("____________________________________________________________\n" +
+                        "No Saved Tasks Found. Let's start with an empty list!\n" +
+                        "____________________________________________________________\n");
+        }
+    }
+
+    public static String stringToSave(ArrayList<Task> listOfTasks) {
```

Comment:
Names representing methods must be verbs and written in camelCase.

maybe use saveToString?